pkg/shell/shell.html

```diff
@@ -1529,6 +1544,58 @@ <h4 class="modal-title" id="account-confirm-delete-title"></h4>
       </div>
     </div>
 
+    <script id="authorized-keys-tmpl" type="x-template/mustache">
+      {{#keys}}
+        <tr>
+        <td>{{comment}}</td>
```

Comment:
Most keys will have a comment usually something like name@machine after the key, that's what shows there if present. See the key used in the tests for an example